inc/classes/class-imagify-views.php

```diff
@@ -214,7 +215,14 @@ public function add_network_menus() {
 	 * @since 1.7
 	 */
 	public function display_settings_page() {
-		$this->print_template( 'page-settings' );
+		$plugin_family = new PluginFamily();
+		$plugin_family = $plugin_family->get_filtered_plugins( 'imagify-plugin/imagify' );
```

Comment:
It's a bit confusing when reading to re-assign the variable from a class instance to the value of the method return.

Maybe rename the second one to `$plugins_array` or something like that?